packages/nx/src/command-line/list.ts

```diff
@@ -58,7 +48,12 @@ export async function listHandler(args: ListArgs): Promise<void> {
     }
     listInstalledPlugins(installedPlugins);
     listCorePlugins(installedPlugins, corePlugins);
-    listCommunityPlugins(installedPlugins, communityPlugins);
+
+    output.note({
+      title:
+        'Nx has a fantastic community that is continuously adding new plugins, tools, and capabilities.',
+      bodyLines: [`Find more great plugins at https://nx.dev/community.`],
```

Comment:
I agree, we do have a fantastic community... but let's say something more like...

What do you think of this?

![image](https://user-images.githubusercontent.com/8104246/235541646-fcda41ac-81b7-4709-86b5-6ba84bc8ce4c.png)

Reply:
Yeah, that could be better. My only hang up is a silly one - I don't really like the way the line lengths increase over the 3 lines. Not got a better solution though, and I like the messaging.

Since its just a bit of bike shedding maybe we could open it up to some more internal people to propose messaging?